stellarphot/tests/test_core.py

```diff
@@ -1015,6 +1041,16 @@ def test_from_vizier_with_coord_and_frame_clip_fails():
         _ = CatalogData.from_vizier(cen_coord, "B/vsx/vsx", clip_by_frame=True)
 
 
+def test_from_vizier_with_header_no_wcs_raise_error():
```

Comment:
I was going to see if you added a test for the no-header situation, glad you did.